managed/ui/src/components/universes/UniverseForm/UniverseFormContainer.js

```diff
@@ -447,7 +452,7 @@ const asyncValidate = (values, dispatch) => {
       values.formType !== 'Async'
     ) {
       dispatch(checkIfUniverseExists(values.primary.universeName)).then((response) => {
-        if (response.payload.status !== 200 && values.formType !== 'Edit') {
+        if (response.payload.status === 200 && values.formType !== 'Edit') {
           reject({ primary: { universeName: 'Universe name already exists' } });
```

Comment:
Error message on 200 status?

Reply:
What was wrong with this approach? It makes sense if you change the function name:
```
dispatch(findUniverseByName(values.primary.universeName)).then((response) => {
     if (response.payload.status === 200 && values.formType !== 'Edit') {
          reject()
```

Reply:
Consider this. Before making any change that I suggested in comments. @gaurav061 @Arnav15

Reply:
Also @jaydeepkumara I was hoping you would consider this before making the changes. I did not tag you because I thought that Gaurav was making these changes.

Reply:
Will following address your concern? (See change to NOT 404 instead of 200.)
```
dispatch(findUniverseByName(values.primary.universeName)).then((response) => {
     if (response.payload.status !== 404 && values.formType !== 'Edit') {
          reject()
```
@jaydeepkumara @Arnav15